docs/user-guide/exploring-dashboards.md

```diff
@@ -12,49 +12,49 @@ In order to create a new dashboard, the **New Dashboard** button <img src="../im
 
 <img src="../img/exploring-dashboards/dashboard-1.jpg" class="ms-docimage"/>
 
-## Topbar
+## Top Toolbar
 
-Through the *Topbar* it is possible to:
+Through the *Top Toolbar* it is possible to:
 
 * Access [GeoSolutions website](https://www.geosolutionsgroup.com/) with a click on the <img src="../img/button/geosolutions-link.jpg" class="ms-docbutton"/> icon
 
 * Set the language, with the *Language switcher*:
 
 <img src="../img/exploring-dashboards/language-switcher.jpg" class="ms-docimage"  style="max-width:150px;"/>
 
-* Go back to the [Homepage](https://mapstore.geosolutionsgroup.com/mapstore/#/) with the <img src="../img/button/home-page-icon.jpg" class="ms-docbutton"/> button
+* Go back to the [Homepage](https://mapstore.geosolutionsgroup.com/mapstore/#/) by clicking the <img src="../img/button/home-page-icon.jpg" class="ms-docbutton"/> button
 
-* Take a look at the account info, change password and logout, with the <img src="../img/button/logged.jpg" class="ms-docbutton"/> button (more info about these options are available in [Managing Users and Groups](managing-users-and-groups.md#managing-users-and-groups) section)
+* **Login/Logout** by clicking the <img src="../img/button/logged.jpg" class="ms-docbutton"/> button (for more information see the [Managing Users and Groups](managing-users-and-groups.md) section)
 
-### Options Menu
+### Side Toolbar
 
-In the **Options** <img src="../img/button/burger.jpg" class="ms-docbutton"/> drop-down menu you can:
+From the **Side Toolbar** the user can:
 
 <img src="../img/exploring-dashboards/options-menu.jpg" class="ms-docimage"  style="max-width:150px;"/>
 
-* **Export** dashboard in `json` format
+* [Add new widgets](adding-widgets.md#adding-widgets) with the <img src="../img/button/++++.jpg" class="ms-docbutton"/> button
 
-* **Import** dashboard in `json` format (it will replace without asking the current dashboard)
+* See the connections between widgets with the <img src="../img/button/show-connections.jpg" class="ms-docbutton"/> button, available when connections are present (more information about this option are available in [Connecting Widgets](connecting-widgets.md#connecting-widgets) section)
 
-* [Save/Save as](resources-properties.md) the dashboard
+* See the **About this dashboard** panel by clicking the <img src="../img/button/details2.jpg" class="ms-docbutton"/> button, when [Details](resources-properties.md#details) are present
 
-* **Delete** the dashboard
+* **Export** dashboard in `json` format by clicking the <img src="../img/button/export2.jpg" class="ms-docbutton"/> button
 
-* Open the [Share](share.md#link) panel
+* **Import** `json` dashboard files from your computer by clicking the <img src="../img/button/import2.jpg" class="ms-docbutton"/> button (it will replace without asking the current dashboard)
```

Comment:
```suggestion
* **Import** `json` dashboard files from your computer by clicking the <img src="../img/button/import2.jpg" class="ms-docbutton"/> button (it will replace the current dashboard directly)
```